### PR TITLE
Refactor the theme build and layout

### PR DESCRIPTION
**Pull request by WadeBarnes**
- Treat ALL themes equally, even the default theme.
- Document the process of how the final theme is constructed by the build.
- Provide some recommendations on how to create and assemble a custom theme.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

### CODE REVIEW

File: client/src/themes/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
...
+
+When creating your own theme create it in a separate folder, [`themes/bcgov`](../../themes/bcgov) for example.  Never mess with the default theme directly.
+
+It is highly recommened you develop your theme in a separate repository and have your automated build process copy the contents of that repository into a named folder in the [`themes`](../../themes) directory of the build's working copy of this repository, and then specifiy the name of that directory in the **THEME** environment variable during the build.  Refer to the `copyCustomTheme` function and it's use in the [manage](../../../docker/manage) script for examples of how this can be accomplished.  For an example of how this can be accomplished in OpenShift with a Jenkins pipeline refer to the [frontend/Jenkinsfile](https://github.com/bcgov/orgbook-configurations/blob/master/jenkins/frontend/Jenkinsfile) in the [bcgov/orgbook-configurations](https://github.com/bcgov/orgbook-configurations) repository.
```
Comment (esune) [inline client/src/themes/README.md:7]: `recommenDed`

**Comment by WadeBarnes** [comment]
This draft was created as a topic for discussion.

Discussion points:
- All themes should be treated as equals.
  - Having the `default` theme in one directory and the *custom* themes in another has caused confusion on how and where a custom theme should be created.
- We want to separate `bcgov/aries-vcr` (the base VCR implementation) from the look at feel, **theme**, of a given instance.  I believe the re-organization and documentation help to make this clear.

File: client/src/themes/README.md (hunk 1 above)
Comment (WadeBarnes) [reply client/src/themes/README.md]: fixed

**Comment by esune** [comment]
The change looks simpler than I initially thought, and it is straightforward enough in the project layout that it should not create confusion.

Wondering if naming the theme that is shipped `base` rather than `default` may help even more driving the idea that it is the base for anything else, and needs to be left alone - but this is kind of nitpicking at this point and probably not necessary.

**Comment by swcurran** [comment]
Definitely agree that base would be better name than default.  Big change?

**Comment by WadeBarnes** [comment]
Super simple change.

**Comment by WadeBarnes** [comment]
Refer to the latest commit to see what it looks like with the `default` theme renamed to `base`.

**Pull request merged**